lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/core/impl/qm/XwikiBridgeInterfaceRestResource.java

```diff
@@ -22,51 +22,61 @@
 import org.xwiki.component.annotation.Component;
 import org.xwiki.rest.XWikiRestComponent;
 
-import eu.learnpad.exception.LpRestException;
-import eu.learnpad.exception.impl.LpRestExceptionImpl;
+import eu.learnpad.exception.impl.LpRestExceptionXWikiImpl;
 import eu.learnpad.qm.BridgeInterface;
 import eu.learnpad.core.rest.RestResource;
 
+/*
+ * The methods inherited form the BridgeInterface in this
+ * class should be implemented as a REST invocation
+ * toward the BridgeInterface binded at the provided URL
+ */
 @Component
```

Comment:
Fine with that, but as you now I am a bit confused now. 
An instance of a Controller must bind a given instance of a bridge (by means of its BridgeInterface), thus there should be a way to say the Controller instance "Hey this (and only this) is the Bridge instance you should use." Is that correct? How to do this if we cannot
relies on constructors?

Also, any Controller should be instantiated only once, and we should have a mean for bind these controller instances, so that Controllers can communicate each-others. Thus an abstract controller (e.g. eu.learnpad.qm.Controller) must have references to all the other controllers it can contact, and there should be a way for instantiating such a binding.  Who is in charge of instantiating a Controller?